src/ReflectionMethod.php

```diff
@@ -107,14 +108,22 @@ public function __toString()
             $this->isFinal() ? ' final' : '',
             $this->isStatic() ? ' static' : '',
             $this->isAbstract() ? ' abstract' : '',
-            join(' ', \Reflection::getModifierNames($this->getModifiers() & 1792)),
+            join(
+                ' ',
+                \Reflection::getModifierNames(
+                    $this->getModifiers() &
+                        (self::IS_PUBLIC | self::IS_PROTECTED | self::IS_PRIVATE)
+                )
+            ),
             $this->getName(),
             $this->getFileName(),
             $this->getStartLine(),
             $this->getEndLine(),
             count($methodParameters),
             $paramString,
-            $hasReturnType ? ReflectionType::convertToDisplayType($this->getReturnType()) : ''
+            (   $returnType ?
+                ReflectionType::convertToDisplayType($returnType) :
+                'UNUSED: Prevents convertToDisplayType() being called.')
```

Comment:
If `UNUSED: Prevents convertToDisplayType() being called.` text isn't actually displayed anywhere, then it doesn't make sense to write it.

If it's displayed, but same text isn't displayed in built-in reflection api, then this won't be compatible.